### PR TITLE
Add createClient

### PR DESCRIPTION
**Pull request by u9g**
kind of does https://github.com/PrismarineJS/bedrock-protocol/issues/35 (still need createServer)

### CODE REVIEW

File: src/createClient.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const { Client } = require('./client')
+const assert = require('assert')
+
+module.exports = { createClient }
+
+/** @param {{ version?: number, hostname: string, port?: number }} options */
+function createClient (options) {
+  assert(options && options.hostname)
+  const client = new Client({ port: 19132, ...options })
```
Comment (rom1504) [inline src/createClient.js:9]: Add port to options

**Comment by rom1504** [comment]
Can you add it to docs ?

**Comment by rom1504** [comment]
API.md

File: src/createClient.js (hunk 1 above)
Comment (u9g) [reply src/createClient.js:9]: it uses rest operator, so iirc it'll just overwrite the port from options if its set

Comment (rom1504) [reply src/createClient.js:9]: Ok

File: docs/api.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Documentation
+
+## be.createServer(options)
```
Comment (rom1504) [inline docs/api.md]: Do you mean createClient ?

Comment (u9g) [reply docs/api.md]: ye, fixed

**Review by rom1504 — APPROVED** [review]

**Pull request merged**